wallet/types/evm_types.go

```diff
@@ -0,0 +1,176 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package types
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/core/types"
+)
+
+// EVMDeployContractInput represents EVM contract deployment parameters
+type EVMDeployContractInput struct {
```

Comment:
We specificy EVMDeployContractInput in case user wants to use Build,Sign,Send Workflow for Deploying contract. We also have Deploy Contract function that does Build,Sign,Send  in one step in local/wallet.go